sys-cluster/spark-bin/spark-bin-2.4.4.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
They look much better! Could you however try building it with EAPI-7 updated here? Namely the eclass should support it...

Reply:
Done - working fine for me. Thank you for all the review.